drivers/windows/windows.go

```diff
@@ -371,6 +367,37 @@ func (d *driver) CreateNetwork(id string, option map[string]interface{}, nInfo d
 
 		config.HnsID = hnsresponse.Id
 		genData[HNSID] = config.HnsID
+		n.created = true
+
+		defer func() {
+			if err != nil {
+				d.DeleteNetwork(n.id)
+			}
+		}()
+
+		hnsIPv4Data := make([]driverapi.IPAMData, len(hnsresponse.Subnets))
+		hnsIPv6Data := make([]driverapi.IPAMData, 0)
```

Comment:
we are not filling any IPV6 data here. why do we need it ?

Reply:
I was just adding if/when we add ipv6 ipam in the future.  I can remove it and add it back when we actually add IPv6 support.

Reply:
got it.  just worried it will confuse sometimes looking at the code down the line.

Reply:
I think it would be better to remove it.  It will create less confusion.  I'll add it back if/when we add ipv6.

Reply:
perfect .  Thanks Pradip